net/spoofer/patches/010-single-threaded.patch

```diff
@@ -0,0 +1,13 @@
+Index: spoofer-1.3.2/openwrt-files/initscript.in
```

Comment:
If you're adding a patch, descriptions of their purpose, like a mailed patch format are very much appreciated.